### DISCUSSION TITLE
Docker Budibase cannot connect to docker Postgresql

### DISCUSSION THREAD

**Discussion by techthrasher** [Budibase/budibase#12679] [Help] [answered]
Hello - I know there have been several posts about this kind of issue, but none of the solutions outlined have helped.

I am running a Budibase docker container on Debian 11, using the installation version of docker-compose.yml, and the .env file with only the secrets changed.  I am also running a Postgresql container, very much out of the box as well.  I can connect to Postgresql with pgAdmin4 all day long but Budibase will not connect at all.   I exposed the Postgres port (5433, because 5432 is already taken) and tried to connect via IP:port; no joy.  I put them on the same docker network and tried to connect via hostname; no joy.  I've set pg to allow all all all trust; no joy.  Absolutely nothing has worked.  

Really like Budibase, but need a relational db for it.  Tried Mariadb, same result...  Don't know if it's Budibase not getting out to the Docker host, or getting out but not able to get into the pg container.  I'm sure it's something fundamental I'm missing...does anyone have a blow by blow of how to set it up?

**Comment by alexcmartins** [comment, marked as answer]
Hello try this host: host.docker.internal:port. 
I was having this same problem and it solved.

**Reply by techthrasher** [reply]
Thank you!!!
